common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -546,6 +539,28 @@
                             self.state.el.find('.video-wrapper').after(self.subtitlesEl);
                             self.state.el.find('.secondary-controls').append(self.container);
                             self.bindHandlers();
+
+                            // add translated instructionals
+                            /* jshint maxlen:300 */
+                            self.subtitlesEl
+                                .attr('aria-label', gettext('Video transcript: activating an item in this group will spool the video to the corresponding time point.'))
+                                .find('.transcript-title')
+                                    .text(gettext('Video transcript'));
+
+                            self.subtitlesEl
+                                .find('.transcript-start')
+                                    .text(gettext('Start of transcript. Skip to the end.'));
+
+                            self.subtitlesEl
+                                .find('.transcript-end')
+                                    .text(gettext('End of transcript. Skip to the start.'));
+
+                            /* jshint maxlen:300 */
+                            self.container
+                                .find('.menu-container .control')
+                                    .attr('aria-label', gettext('Language: Press the UP arrow key to enter the language menu then use UP and DOWN arrow keys to navigate language options. Press ENTER to change to the selected language.'))
+                                    .find('.control-text')
+                                        .text(gettext('Open langage menu.'));
                         }
```

Comment:
The i18n didn't want to work in the earlier object that was $.join'ed, but after looking at how other files handle this, I've moved these here to be updated later.